context-free.md

```diff
@@ -2,21 +2,54 @@
 
 ##  Who is this programming language for?
 
+This programming language is for anyone who wants to create images using written instructions.
 
 ## What is easy to do in this language? Why is it easy?
 
+It's easy to make recursive, fractal-like patterns! 
+It's also very easy to change the location, size, color, hue, angle, 
+and saturation of any object. 
+
+The syntax of this language was also straightforward. 
+It seems that if the user does not specify a 
+location, size, color, hue, angle, or saturation of an object, 
+the program knows to set the default. 
+Thus, one only has to explicitly state if he/she wants to; 
+for example, if one only wants to change the color of an object
+from the default color to another color, 
+and does not have to explicitly specify keeping other elements the default.
+
+Also, it is nice that writing "s 6 b 0.5" is the same thing as "b 0.5 s 6 b". 
+I appreciated this flexibility in specifying the features of an object.
 
 ## What is hard to do in this language? Why is it hard?
 
+It was hard to make images that look like letters -- 
+maybe there is a shortcut I did not realize, but it was a little annoying to 
+have to use lines and arcs to create letters, instead of being able to 
+type out the letters themselves.
```

Comment:
Yep - this was a design decision, and raises a few questions: "What were the designers trying to make me do? If this is hard, am I doing something they didn't want, or something they didn't plan for? Is that a problem?".